libs/hdf-converters/test/attestations/attestations.spec.ts

```diff
@@ -6,108 +6,371 @@ import {
   advanceDate,
   Attestation,
   createAttestationMessage,
-  parseXLSXAttestations
+  parseXLSXAttestations,
+  convertAttestationToSegment
 } from '../../src/utils/attestations';
 import {omitHDFTimes, omitVersions} from '../utils';
-
-const attestationArray: Attestation[] = [
+import _ from 'lodash';
+// test a new attestation array for each frequency, use arbitrary far date (yr 3000, 1900)
+// try applying to attestations w diff statuses
+const attestationMessageArray: Attestation[] = [
```

Comment:
Is it easier to also break these down into individual attestation arrays of one item with more meaningful names? When you call `attestationMessageArray[0]` in the tests, it doesn't tell me so much.

Reply:
Switching to not use this array of attestations entirely in favor of the more clearly named individual attestations below